pkg/fleet/installer/bootstrap/bootstrap_windows.go

```diff
@@ -75,18 +76,22 @@ func downloadInstaller(ctx context.Context, env *env.Env, url string, tmpDir str
 		return nil, fmt.Errorf("failed to extract layers: %w", err)
 	}
 
-	msis, err := filepath.Glob(filepath.Join(tmpDir, "datadog-agent-*-x86_64.msi"))
+	installPath, err = getInstallerFromMSI(tmpDir)
 	if err != nil {
-		return nil, err
-	}
-	if len(msis) > 1 {
-		return nil, fmt.Errorf("too many MSIs in package")
-	} else if len(msis) == 0 {
-		return nil, fmt.Errorf("no MSIs in package")
+		installPath, err = getInstallerFromOCI(tmpDir)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get installer: %w", err)
+		}
 	}
+	return iexec.NewInstallerExec(env, installPath), nil
+}
 
+func getInstallerFromMSI(tmpDir string) (string, error) {
+	msis, err := filepath.Glob(filepath.Join(tmpDir, "datadog-agent-*-x86_64.msi"))
```

Comment:
[filepath.Glob docs](https://pkg.go.dev/path/filepath#Glob) say it returns `nil`, not an error when there are no matching files. this needs to be explicitly checked, else I think `msis[0]` will panic.